apps/web/src/app/error.tsx

```diff
@@ -0,0 +1,48 @@
+'use client';
+
+import React, { useEffect } from 'react';
+import { Button } from '@hypha-platform/ui';
+import { ReloadIcon } from '@radix-ui/react-icons';
+
+export default function AppError({
+  error,
+  reset,
+}: {
+  error: Error & { digest?: string };
+  reset: () => void;
+}) {
+  useEffect(() => {
+    console.error(error);
+  }, [error]);
```

Comment:
Does effect really necessary here? I think logging error on render would fit just fine.
```suggestion
  console.error(error);
```